Source/WebCore/bindings/js/JSDOMGlobalObject.cpp

```diff
@@ -475,14 +475,7 @@ static JSC::JSPromise* handleResponseOnStreamingAction(JSC::JSGlobalObject* glob
     }
 
     auto body = inputResponse->consumeBody();
-    WTF::switchOn(body, [&](Ref<FormData>& formData) {
-        if (auto buffer = formData->asSharedBuffer()) {
-            compiler->addBytes(buffer->data(), buffer->size());
-            compiler->finalize(globalObject);
-            return;
-        }
-        // FIXME: Support FormData loading.
-        // https://bugs.webkit.org/show_bug.cgi?id=221248
+    WTF::switchOn(body, [&](Ref<FormData>&) {
         compiler->fail(globalObject, createDOMException(*globalObject, Exception { NotSupportedError, "Not implemented"_s  }));
```

Comment:
Is “not implemented” still the correct exception here? I am a little puzzled about removing the comment that says this needs to be implemented, but still keeping a “not implemented” exception.

Reply:
We can make it RELEASE_ASSERT_NOT_REACHED.

Reply:
And changed ;)